stage0/layout.ld

```diff
@@ -275,7 +272,7 @@ SECTIONS {
      * Unfortunately it doesn't look like `SIZEOF()` works here, so we have to keep track of the data
      * structure size by hand to ensure that it ends at the correct location in memory.
      */
-    .guid_tables TOP - 0x20 - (16 + 4*12 + 2*22 + 18) : {
+    .guid_tables TOP - 0x20 - (16 + 3*12 + 2*22 + 18) : {
```

Comment:
Why did this change?

Reply:
We used to have four entries in the GUIDed tables; this PR deletes the one for the zero page, thus we're down to three entries.